sources/engine/Stride.BepuPhysics/Stride.BepuPhysics/Constraints/TwistLimitConstraintComponent.cs

```diff
@@ -15,7 +15,12 @@ namespace Stride.BepuPhysics.Constraints;
 [ComponentCategory("Physics - Bepu Constraint")]
 public sealed class TwistLimitConstraintComponent : TwoBodyConstraintComponent<TwistLimit>, ISpring
 {
-    public TwistLimitConstraintComponent() => BepuConstraint = new() { SpringSettings = new SpringSettings(30, 5) };
+    public TwistLimitConstraintComponent() => BepuConstraint = new()
```

Comment:
I must say, I'm not a fan of the lambda syntax for methods in general and even less for constructors. IMO it should be reserved for property accessors on in rare case when a method redirect to another (e.g. for wrapping types).

Reply:
I used them for small methods like these which are easy to read and it removes a few lines. I mean, the readability is still ok 🙂.

Reply:
I guess I'm not used to it.

Reply:
I'm not too keen on it either, but felt that in this case it was warranted as those classes are effectively just wrappers around bepu's constraints. If I were to change that, I would have to change the ~30 other constraint classes, let me know.

Reply:
It was just a remark on my side. I guess it's a personal preference and it doesn't warrant a change in this PR